### DISCUSSION TITLE
Connectivity vs Process Paste Status

### DISCUSSION THREAD

**Discussion by kendal-kirby** [twilio-labs/paste#2113] [Office hours] [answered]
I am in the process of redesigning the [Twilio Status page](https://status.twilio.com/) (which is currently not using Paste) and I'd like some clarity on when to use the connectivity status icons (dots) vs the process status icons (symbols). The status dot/symbol will only be visible for non-operational states. 

The possible statuses that can appear are (with proposed status indicators):
- Operational (text only)
- Degraded performance (text + orange)
- Partial outage (text + orange)
- Major outage (text + red)
- Under maintenance (text + blue)

[See wireframes](https://www.figma.com/file/2GCv7TlUTudQ2ihkpMLnbt/Status-Page?node-id=12%3A3916)

**Comment by loreina** [comment]
Hi @kendal-kirby,

Connectivity status icons are used to indicate if a user or system is online or offline. Process status icons are used to indicate stages in a process. There's some information on our doc site with more detail about the differences: https://paste.twilio.design/patterns/status/#variations

For the possible statuses, could you share more about mixing plain text statuses with text+icon statuses? I see that Operational is text only, and am wondering if on the status page where there are multiple statuses displaying, that the plain text status might look visually different than the rest. 

Let us know if there's anything else we can help with!

**Reply by kendal-kirby** [reply]
Thanks for all the help here and during office hours!

As discussed during office hours, I'd like to make a request to add one more connectivity status for neutral statuses (like in the process status pattern).

**Comment by kate-porter** [comment, marked as answer]
Hey @kendal-kirby! Thanks for coming to office hours last week. [Here's a link to the recording](https://www.loom.com/share/5bf8656ecbaa4234a2e96ffcc411c86e) for future reference (password is shared in the #help-design-system Slack channel).

I'll share your request to add a new neutral connectivity status icon with the team and report back!

**Reply by kate-porter** [reply]
Hey @kendal-kirby - chatted with the team about adding a new neutral connectivity status, and we've all agreed that it makes sense to do so. We'll work on getting this added within our next couple of sprints. What's your timeline look like for when you'll need this?

**Reply by kendal-kirby** [reply]
@katieporter The timeline is still TBD, but development likely won't begin for at least another month.

**Comment by kate-porter** [comment]
Hey @kendal-kirby! We just published an update to the Paste Patterns library that includes the new Neutral variant for Connectivity Statuses. We're aiming to get it added to the Paste codebase and docs site in the next sprint or two.

**Reply by kendal-kirby** [reply]
Thanks for the update @katieporter!

**Reply by shleewhite** [reply]
Hi @kendal-kirby! I just merged [a PR to add the ConnectivityNeutralIcon](https://github.com/twilio-labs/paste/pull/2215) and include it in the Status pattern page. The updates should be available soon!
